client/src/pages/RequestDashboardPage/RequestDashboard.jsx

```diff
@@ -7,21 +6,17 @@ const RequestDashboard = () => {
       <h1 className="text-3xl">Referral Request</h1>
 
       <div className="w-full grid grid-cols-2 gap-10">
-        <RequestCard company="Google" priority={1} status="waiting"/>
-        <RequestCard company="Amazon" priority={1} status="waiting"/>
-        <RequestCard company="Meta" priority={1} status="approved"/>
-        <RequestCard company="LinkedIn" priority={1} status="referred"/>
-        <RequestCard company="Netflix" priority={1} status="waiting"/>
-        <RequestCard company="Microsoft" priority={1} status="matched"/>
-        <RequestCard company="DoorDash" priority={1} status="waiting"/>
-        <RequestCard company="Vanta" priority={1} status="referred"/>
-        <RequestCard company="Tesla" priority={1} status="waiting"/>
+        <RequestCard company="Google" priority={1} status="waiting" />
+        <RequestCard company="Amazon" priority={1} status="waiting" />
+        <RequestCard company="Meta" priority={1} status="approved" />
+        <RequestCard company="LinkedIn" priority={1} status="referred" />
+        <RequestCard company="Netflix" priority={1} status="waiting" />
+        <RequestCard company="Microsoft" priority={1} status="matched" />
+        <RequestCard company="DoorDash" priority={1} status="waiting" />
+        <RequestCard company="Vanta" priority={1} status="referred" />
+        <RequestCard company="Tesla" priority={1} status="waiting" />
```

Comment:
could add a CI/CD check on this to ensure it's always formatted properly

Reply:
yeah, em cũng tính sẽ làm cái đấy cho đỡ phải style thiếu. Thank you anh đã review <3